src/storage/src/hummock/shared_buffer/shared_buffer_batch.rs

```diff
@@ -93,11 +80,41 @@ impl SharedBufferBatch {
     pub fn new(
```

Comment:
We may rename it to `for_test` to avoid future misuse.

Reply:
fixed.